machine/reference/env.md

```diff
@@ -46,7 +46,7 @@ $ # The environment variables have been unset.
 The output described above is intended for the shells `bash` and `zsh` (if
 you're not sure which shell you're using, there's a very good possibility that
 it's `bash`). However, these are not the only shells which Docker Machine
-supports. Depending of the environment you're running your command into we will print them for the proper system.
+supports. Depending of the environment you're running your command into, we will print them for the proper system.
```

Comment:
This sentence could use some rewriting. Maybe something like "Docker Machine detects the shells available in your environment and lists them."